### PR TITLE
next-pylint weekly pipeline

### PR DESCRIPTION
**Pull request by l0lawrence**
Add next-pylint features to test-weekly, will merge into upstream branch and then merge that branch into main

### CODE REVIEW

File: eng/tox/tox.ini [hunk 1]
```diff
@@ -39,7 +39,6 @@ pkgs =
     wheel==0.37.0
     packaging==20.4
     urllib3==1.26.12
-    tomli==2.0.1
```
Comment (scbedd) [inline eng/tox/tox.ini:42]: Revert this as well plox.

**Comment by l0lawrence** [comment]
azp run python - eventgrid - tests-weekly

**Pull request closed without merging**